diecast

```diff
@@ -1 +0,0 @@
-cmd/diecast
```

Comment:
What was this? Sure it's not actually doing anything? #mysteries #garyverse

Reply:
that was a symlink to a compiled version of this project. Totally unnecessary and should never have been checked in to begin with.